### PR TITLE
README refactor and indexes

### PR DESCRIPTION
**Pull request by bseb**
### Background
README has some outdated links,  added some indexes to make it easer to find detections.

### Changes
Added indexes
Updated some Links
New Logo ( text was hard to see in GitHub dark mode)

### Testing
Unit tests, but this is all Markdown changes

### CODE REVIEW

**Comment by darwayne** [comment]
noice

File: README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
 <p align="center">
-  <a href="https://www.runpanther.io"><img src=".img/panther-logo-github-highres.png" width=75% alt="Panther Logo"/></a>
+  <a href="https://www.runpanther.io"><img src=".img/panther-logo-github-highres.png" width=40% alt="Panther Logo"/></a>
```
Comment (k-bailey) [inline README.md:2]: idk if this is really necessary but should we move all of these links to panther.com?

File: README.md [hunk 2]
```diff
@@ -21,26 +20,55 @@ Panther is a security analytics platform built for cloud-focused security teams.
 
 Panther enables teams to define detections as code and programmatically upload them to your Panther deployment.
 
-This repository contains all of Panther's built-in detections  installed by default.
+This repository contains all of Panther created detections that can be added to your Panther instance.
```
Comment (k-bailey) [inline README.md:23]: maybe remove of? I can't tell if it is just me but this reads a little strange

File: indexes/gcp.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+[ GCS Bucket Made Public](../gcp_audit_rules/gcp_gcs_public.py)
+
+[ GCP Resource in Unused Region](../gcp_audit_rules/gcp_unused_regions.py)
+
+[ GCP Resource in Unused Region](../gcp_audit_rules/gcp_unused_regions.py)
```
Comment (k-bailey) [inline indexes/gcp.md:5]: duplicate

File: README.md (hunk 1 above)
Comment (bseb) [reply README.md]: Good catch,  updated along with a couple doc links I missed

File: README.md [hunk 4]
```diff
@@ -21,71 +20,91 @@ Panther is a security analytics platform built for cloud-focused security teams.
 
 Panther enables teams to define detections as code and programmatically upload them to your Panther deployment.
 
-This repository contains all of Panther's built-in detections  installed by default.
+This repository contains all the detections developed by Panther for use in your Panther deployment.
```
Comment (bseb) [inline README.md:23]: @k-bailey  is this any better? I agree that this read weird before. but maybe it still does

Comment (bseb) [reply README.md:23]: I checked in with Sally S and she confirmed that this updated version reads ok. I think this is ready to ship

File: README.md (hunk 2 above)
Comment (bseb) [reply README.md]: See updated comments

**Review by k-bailey — APPROVED** [review]

**Pull request merged**